src/value.rs

```diff
@@ -583,30 +675,37 @@ impl Value {
             _ => panic!("not Value::DateTime"),
         }
     }
-    #[cfg(not(feature = "with-chrono"))]
+    #[cfg(feature = "with-time")]
+    pub fn as_ref_date_time(&self) -> Option<&PrimitiveDateTime> {
+        match self {
+            Self::DateTime(v) => box_to_opt_ref!(v),
+            _ => panic!("not Value::DateTime"),
+        }
+    }
+    #[cfg(not(any(feature = "with-chrono", feature = "with-time")))]
     pub fn as_ref_date_time(&self) -> Option<&bool> {
         panic!("not Value::DateTime")
     }
 }
 
 impl Value {
     pub fn is_date_time_utc(&self) -> bool {
-        #[cfg(feature = "with-chrono")]
+        #[cfg(any(feature = "with-chrono", feature = "with-time"))]
         return matches!(self, Self::DateTimeUtc(_));
-        #[cfg(not(feature = "with-chrono"))]
+        #[cfg(not(any(feature = "with-chrono", feature = "with-time")))]
         return false;
     }
```

Comment:
I think we could refactor all these feature guards by adding a marker feature, perhaps named `_with-chrono-or-time`. This feature will be turned on if either `with-chrono` or `with-time` was enabled.

Then, we could simple check
```rust
#[cfg(feature = "_with-chrono-or-time")]
#[cfg(not(feature = "_with-chrono-or-time"))]
```

Reply:
I think it's fine either way